packages/blocks/aws/src/lib/actions/cli/aws-cli.ts

```diff
@@ -1,10 +1,19 @@
 import { runCliCommand } from '@openops/common';
+import { SharedSystemProp, system } from '@openops/server-shared';
 
 export async function runCommand(
   command: string,
   region: string,
   credentials: any,
 ): Promise<string> {
+  if (
+    credentials.accessKeyId === undefined &&
```

Comment:
might be better to do !accessKeyId to deal with empty string too and null.
Also add tests